plugins/woocommerce/package.json

```diff
@@ -261,7 +261,9 @@
 					"changes": [
 						"tests/e2e-pw/**"
 					],
-					"onlyForDependencies": ["@woocommerce/e2e-utils-playwright"],
+					"onlyForDependencies": [
+						"@woocommerce/e2e-utils-playwright"
+					],
```

Comment:
I see changes in formatting. It is aligned with the other values around. It looks like it fixes previously incorrect formatting. So I think this is fine.

Reply:
Yes, this was done automatically (either by pnpm or one of its commands).